recipes/sparsehash-c11/meta.yaml

```diff
@@ -0,0 +1,62 @@
+{% set name = "sparsehash-c11" %}
+# Using same version as last `sparsehash` version, from which this code is based
+{% set version = "2.0.3" %}
+# Using a commit_ref because there is no version tag
```

Comment:
I've opened an [issue upstream](https://github.com/sparsehash/sparsehash-c11/issues/24) for this

Reply:
@ocefpaf the request to add a tag wasn't added... do you thing it's ok to version it like this, or should I use a different thing like a hash or a `0.something` version?

Reply:
B/c it is tied to the `sparsehash` version this is OK IMO. Also, looks like there won't be an official release of these headers anytime soon. If such thing happen we can pull these packages and do it properly when upstream issue a release.